tests/null_queries/tests.py

```diff
@@ -39,9 +39,9 @@ def test_none_as_null(self):
         with self.assertRaises(ValueError):
             Choice.objects.filter(id__gt=None)
 
-        # Related managers use __exact=None implicitly if the object hasn't been saved.
         p2 = Poll(question="How?")
-        self.assertEqual(repr(p2.choice_set.all()), '<QuerySet []>')
+        with self.assertRaises(ValueError):
```

Comment:
Use `assertRaiseMessage` as in the other tests (maybe this is better with the other added tests though).